server.ts

```diff
@@ -102,11 +104,30 @@ if (isTestInstance()) {
     credentials: true
   }))
 }
-
 // For the logger
-app.use(morgan('combined', {
-  stream: { write: logger.info.bind(logger) }
-}))
+app.use(morgan(
+  (tokens, req, res) => { // this is the programmatic equivalent to Apache's combined logging
```

Comment:
Well ^^

Maybe we could do this with a special morgan token: https://github.com/expressjs/morgan#tokens and use it in a [combined format](https://github.com/expressjs/morgan#combined)